Unix/tools/allocdump

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+SCRIPTDIR=$(dirname "$0")
+BINDIR=/opt/omi/bin
+OMISERVER=${BINDIR}/omiserver
+GDB=gdb
+GDB_SCRIPT="$SCRIPTDIR/gdb_allocdump"
+MINIMUM_VERSION="100401000"   # version 1.4.1-0
+CONFIGEDITOR=${BINDIR}/omiconfigeditor
+CONF_FILE=/etc/opt/omi/conf/omiserver.conf
+
+# Verify that the GDB script file exists
+if [ ! -f ${GDB_SCRIPT} ]; then
+    echo "Gdb script file ${GDB_SCRIPT} does not exist"
+    exit 1
+fi
+
+# Verify that gdb exists
+if [ ! -x "$(command -v ${GDB})" ]; then 
+    echo "${GDB} not found in your PATH"
+    exit 1
+fi
+
+# Verify sudo
+EFFECTIVE_ID="$(id -u)"
+SUDO=
+if [ ${EFFECTIVE_ID} -ne 0 ]; then
+    SUDO=sudo
+    if [ ! -x "$(command -v ${SUDO})" ]; then 
+        echo "${SUDO} not found in your PATH"
+        exit 1
+    fi
+    ${SUDO} -v >&1 >/dev/null 
+    if [ $? -eq 1 ]; then
+        echo "You do not have access to run ${SUDO}"
+        exit 1
+    fi
+fi
+
+# Verify that omiserver process is running
+ps -ef | fgrep ${OMISERVER} | fgrep -v fgrep 2>&1 >/dev/null
+if [ $? -eq 1 ]; then
+    echo "Omiserver is not running"
+    exit 1
+fi
+
+# Verify that omiserver.conf is set up properly
+if [ ! -f ${CONF_FILE} ]; then 
+    echo "${CONF_FILE} does not exist"
+    exit 1
+fi
+if [ ! -x ${CONFIGEDITOR} ]; then 
+    echo "${CONFIGEDITOR} does not exist"
+    exit 1
+fi
+for i in INFO DEBUG VERBOSE; do 
+    ${CONFIGEDITOR} loglevel -q $i < ${CONF_FILE} 2>&1 >/dev/null; 
+    [ $? -eq 0 ] && found=1 && break; 
+done
+if [ "$found" != "1" ]; then
```

Comment:
`found` isn't initialized. That's fine, but perhaps this should be a little more unique, like `loglevel_okay`?

Reply:
Changed name to loglevel_okay.